### PR TITLE
Ported some sensor files

### LINKED ISSUE #64: port MPU6050.py

### LINKED ISSUE #65: Port MF522.py

### PR DESCRIPTION
**Pull request by akarshan96**
- Created MPU6050.java, fixes #65 
- Created MF522.java, fixes #64 
- Added Filters (Kalman Filter and Complementary Filter)
- Added Apache Commons Maths Library for scientific calculations

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/communication/sensors/MPU6050.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                }
+                return raw;
+            }
+        }
+        return null;
+    }
+
+    public double[] getAcceleration() throws IOException {
+        ArrayList<Character> vals = getVals(0x3B,6);
+        int ax = int16(vals.get(0) << 8 | vals.get(1));
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/MPU6050.java]: I am not sure if this does anything finally.
you are giving and "int" to int16 to convert it to short and then again putting it into "int". So number of bits are same as before.
remove short, we would work with "int" everywhere. As it just a matter of number of bits and Lower bit order is same in both short and int.

P.S: when implemented something just do a dry run to get to know what its doing.

Comment (akarshan96) [inline app/src/main/java/org/fossasia/pslab/communication/sensors/MPU6050.java]: @viveksb007 the suggested are made

**Comment by akarshan96** [comment]
@viveksb007 the suggested are made

**Comment by akarshan96** [comment]
@mariobehling thanks for the update :)

**Pull request merged**